### PR TITLE
ROC-6537: Recreate direction order pdf during direction order

### PR DESCRIPTION
**Pull request by dharmendrak**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-6537

### Change description ###

ROC-6537: Recreate direction order pdf during direction order

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/DocAssemblyService.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        this.docAssemblyTemplateBodyMapper = docAssemblyTemplateBodyMapper;
+        this.docAssemblyClient = docAssemblyClient;
+        this.userService = userService;
+        this.templateId = templateId;
+    }
+
+    public DocAssemblyResponse createOrder(CCDCase ccdCase, String authorisation) {
+        UserDetails userDetails = userService.getUserDetails(authorisation);
+
+        logger.info("Generate order callback: creating request for doc assembly");
```
Comment (simad) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/DocAssemblyService.java]: this is not really a callback anymore, so i'd reword the log line

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/DocAssemblyService.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        DocAssemblyRequest docAssemblyRequest = DocAssemblyRequest.builder()
+            .templateId(templateId)
+            .outputType(OutputType.PDF)
+            .formPayload(docAssemblyTemplateBodyMapper.from(ccdCase, userDetails))
+            .build();
+
+        logger.info("Generate order callback: sending request to doc assembly");
```
Comment (simad) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/DocAssemblyService.java]: same as above, not a callback so the log is misleading

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/DrawOrderCallbackHandler.java [hunk 3]
```diff
@@ -40,25 +42,32 @@
 @Service
 @ConditionalOnProperty(prefix = "document_management", name = "url")
```
Comment (daveoh) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/DrawOrderCallbackHandler.java]: Should be updated doc_assembly

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/DocAssemblyService.java (hunk 2 above)
Comment (dharmendrak) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/DocAssemblyService.java]: done

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/DocAssemblyService.java (hunk 1 above)
Comment (dharmendrak) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/DocAssemblyService.java]: done
